### DISCUSSION TITLE
Subtable Members are all null after reading the table

### DISCUSSION THREAD

**Discussion by febus** [linq2db/linq2db#3677] [Q&A] [answered]
Hello
linq2db CLI generates also members for sub-tables. For example Customer has many Orders.

The problem is, all sub-table members are null at the runtime, after the main table has been loaded from the database. 
Actually there are some entries in the sub-table, so that  sub-table should actually be filled with values.
Should I do something in code to fill sub-tables of the "master" object? I've been looking for someting like "Include", but without any success.
Thanks in advance!

**Comment by MaceWindu** [comment, marked as answer]
Yes, you need to specify loaded dependencies explicitly using `LoadWith/ThenLoad` methods.
Check tests for some examples https://github.com/linq2db/linq2db/blob/master/Tests/Linq/Linq/EagerLoadingTests.cs
